modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards_admin/js/BanUsersManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {postForm} from 'frontend-js-web';
+
+export default function propsTransformer({
+	additionalProps: {banUsersURL},
+	portletNamespace,
+	...otherProps
+}) {
+	const unbanUser = () => {
```

Comment:
Since there's only one function, we could inline it, at least that's what I was asked to do [here](https://github.com/liferay-frontend/liferay-portal/pull/761#discussion_r574407149),